packages/governance/src/multiPluralityVoteCounter.js

```diff
@@ -0,0 +1,242 @@
+
+  let isOpen = true;
+  const positions = questionSpec.positions;
+
+  /** @type { PromiseRecord<Position | Position[]> } */
+  const outcomePromise = makePromiseKit();
+  /** @type { PromiseRecord<VoteStatistics> } */
+  const tallyPromise = makePromiseKit();
+
+  /**
+   * @typedef {object} RecordedBallot
+   * @property {Position} chosen
+   * @property {bigint} shares
+   */
+  /** @type {Store<Handle<'Voter'>,RecordedBallot> } */
+  const allBallots = makeStore('voterHandle');
+
+  const countVotes = () => {
+    assert(!isOpen, X`can't count votes while the election is open`);
+
+    let spoiled = 0n;
+    const tally = Array(positions.length).fill(0n);
+
+    for (const { chosen, shares } of allBallots.values()) {
+      const positionIndex = positions.findIndex(p => keyEQ(p, chosen));
+      if (positionIndex < 0) {
+        spoiled += shares;
+      } else {
+        tally[positionIndex] += shares;
+      }
+    }
+
+    /** @type { VoteStatistics } */
+    const stats = {
+      spoiled,
+      votes: allBallots.getSize(),
+      results: tally.map((_, i) => ({
+        position: positions[i],
+        total: tally[i],
+      })),
+    };
+
+    tallyPromise.resolve(stats);
+
+    if (!makeQuorumCounter(threshold).check(stats)) {
+      outcomePromise.reject('No quorum');
+      /** @type {OutcomeRecord} */
+      const voteOutcome = {
+        question: details.questionHandle,
+        outcome: 'fail',
+        reason: 'No quorum',
+      };
+      E(publisher).publish(voteOutcome);
+      return;
+    }
+
+    const sortedPositions = stats.results.sort((a, b) => {
+      if (a.total < b.total) return 1;
+      if (a.total > b.total) return -1;
+      return 0;
+    });
+
+    const winningPositions = [];
+    for (const position of sortedPositions) {
+      if (position.total > 0n) {
+        if (winningPositions.length < questionSpec.maxWinners) {
+          winningPositions.push(position);
+        } else if (
+          winningPositions[winningPositions.length - 1].total === position.total
+        ) {
+          winningPositions.push(position);
+        }
+      }
+    }
+
+    if (winningPositions.length === 0) {
+      outcomePromise.resolve(questionSpec.tieOutcome);
+    } else if (winningPositions.length <= questionSpec.maxWinners) {
+      const outcome = winningPositions.map(p => p.position);
+      outcomePromise.resolve(outcome);
+    } else {
+      const tieIndex = questionSpec.maxWinners - 1;
+      const untiedPositions = winningPositions.slice(0, tieIndex);
+
+      const tiedPositions = winningPositions.slice(tieIndex);
+      const tieWinner = tiedPositions[crypto.randomInt(tiedPositions.length)];
+
+      const outcome = untiedPositions.concat(tieWinner).map(p => p.position);
+      outcomePromise.resolve(outcome);
+    }
+
+    E.when(outcomePromise.promise, position => {
+      /** @type { OutcomeRecord } */
+      const voteOutcome = {
+        question: details.questionHandle,
+        position,
```

Comment:
`position` could be `positions` or `outcome`. I suggest changing `OutcomeRecord`'s `position` to be `positions`.

```suggestion
    E.when(outcomePromise.promise, outcome => {
      /** @type { OutcomeRecord } */
      const voteOutcome = {
        question: details.questionHandle,
        positions: outcome,
```